mocks/README.md

```diff
@@ -0,0 +1,140 @@
+# Mockserver
+
+The mock server allows you to test any of the exporters against mockup endpoints of different sources (github, gitlab, jira, etc)
+
+## Start up the mock server
+
+The mock server can be started using the [Mockoon GUI](https://mockoon.com/docs/latest/gui-cheat-sheet/) or the [Mockoon CLI](https://github.com/mockoon/cli#installation) or using a container with the [Mockoon CLI](https://hub.docker.com/r/mockoon/cli).
+
+Let's start the mock server using the container approach.
+
+First, you will need to install a container engine like [Podman](https://podman.io/)
+
+Then clone this repository and execute the following command to startup the mock server.
+
+```sh
+$ git clone https://github.com/konveyor/pelorus.git
+```
+
+```sh
+$ podman run --name mockoon --rm -v <path-to-repo>/pelorus/mocks/commitexporter_github.json:/data:z -p 3000:3000 mockoon/cli:latest -d data -i 0
+
+Mock started at https://localhost:3000 (pid: 0, name: mockoon-openshift)
+```
+
+### Test against the mock server
+
+Set up your envs in order to use the mock server.
+
+```sh
+export GIT_USER=gituser
+export GIT_TOKEN=gittoken
+export GIT_API=localhost:3000
+export LOG_LEVEL=DEBUG
+export NAMESPACES=basic-nginx-build,basic-nginx-dev,basic-nginx-stage,basic-nginx-prod
+export TLS_VERIFY=False
+```
+---
+**NOTE**
+
+The env TLS_VERIFY is needed because the mocks server use a self-signed certificate.
+
+---
+
+Because the commit exporters search on Openshift the information from the build, first you need to execute "oc login" against the mock server
```

Comment:
I want to make sure I understand-- this uses mockoon's proxy mode? I think we should document that, and note that the server to proxy to is hardcoded in mockoon-- do we have some way of templating that out?

Reply:
Is not using mockoon on proxy mode, when you start the mock server that is disable, its only need to be enable when your are creating new scenarios but this is not the case

Reply:
So then this isn't necessary for setup, right? Only for development? Then maybe we should document it elsewhere?

Reply:
To start the mock server, they are not necessary, but in this case, the instructions are to test an exporter against the mock server, in that case, you need to set up the env vars and login into the server as if you were using Openshift instance.